### ISSUE TITLE
Create just Metadata

### ISSUE THREAD

**Issue by Frexxd** [rounakbanik/generative-art-nft#29]
Is it possible to just create metadata json files?

I have 10000 images all loaded on piñata.
I don’t require a ‘traits’ or ‘attributes’ section in the metadata.

**Comment by rounakbanik** [comment]
Yes. You will have to remove the code that generates trait and attributes metadata.
